pkg/controlplane/apiserver/options/options.go

```diff
@@ -202,6 +209,13 @@ func (s *Options) AddFlags(fss *cliflag.NamedFlagSets) {
 
 	fs.StringVar(&s.ServiceAccountSigningEndpoint, "service-account-signing-endpoint", s.ServiceAccountSigningEndpoint, ""+
 		"Path to socket where a external JWT signer is listening. This flag is mutually exclusive with --service-account-signing-key-file and --service-account-key-file. Requires enabling feature gate (ExternalServiceAccountTokenSigner)")
+
+	fs.DurationVar(&s.CoordinatedLeadershipLeaseDuration, "coordinated-leadership-lease-duration", s.CoordinatedLeadershipLeaseDuration,
```

Comment:
You could also add validation for these flags under https://github.com/kubernetes/kubernetes/blob/master/pkg/controlplane/apiserver/options/validation.go and have corresponding tests in pkg/controlplane/apiserver/options/validation_test.go if it makes sense